modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/rebalance/DistributionZoneRebalanceEngine.java

```diff
@@ -334,6 +342,25 @@ public void onError(Throwable e) {
         };
     }
 
+    private int getCatalogVersionForCounter(int zoneId, int partId, long revision) {
```

Comment:
Please add @Deprecated for removal here with collocation epic https://issues.apache.org/jira/browse/IGNITE-22115